package.json

```diff
@@ -28,34 +27,40 @@
     "url": "https://github.com/Microsoft/powerbi-visuals-gantt/issues"
   },
   "homepage": "https://github.com/Microsoft/powerbi-visuals-gantt#readme",
-  "devDependencies": {
-    "@types/jasmine": "2.5.37",
-    "@types/jasmine-jquery": "1.5.28",
-    "coveralls": "2.11.15",
+  "dependencies": {
     "d3": "3.5.5",
     "globalize": "0.1.0-a2",
-    "jasmine": "2.5.2",
-    "jasmine-jquery": "2.1.1",
     "jquery": "3.1.1",
-    "karma": "1.3.0",
+    "lodash": "4.16.2",
+    "powerbi-visuals-utils-chartutils": "0.2.1",
+    "powerbi-visuals-utils-colorutils": "0.2.2",
+    "powerbi-visuals-utils-dataviewutils": "1.1.1",
+    "powerbi-visuals-utils-formattingutils": "1.0.0",
+    "powerbi-visuals-utils-interactivityutils": "0.3.1",
+    "powerbi-visuals-utils-svgutils": "0.2.1",
+    "powerbi-visuals-utils-tooltiputils": "0.3.2",
+    "powerbi-visuals-utils-typeutils": "1.0.0"
+  },
+  "devDependencies": {
+    "@types/d3": "3.5.38",
+    "@types/jasmine": "2.5.47",
+    "@types/jasmine-jquery": "1.5.30",
+    "@types/jquery": "2.0.41",
+    "@types/lodash": "4.14.55",
+    "coveralls": "2.13.0",
+    "jasmine": "2.5.3",
+    "jasmine-jquery": "2.1.1",
+    "karma": "1.6.0",
     "karma-chrome-launcher": "2.0.0",
     "karma-coverage": "1.1.1",
-    "karma-jasmine": "1.0.2",
-    "karma-remap-istanbul": "0.4.0",
+    "karma-jasmine": "1.1.0",
+    "karma-remap-istanbul": "0.6.0",
     "karma-sourcemap-loader": "0.3.7",
-    "karma-typescript-preprocessor": "0.3.0",
-    "lodash": "4.16.2",
-    "powerbi-visuals-tools": "1.4.3",
-    "powerbi-visuals-utils-colorutils": "^0.2.1",
-    "powerbi-visuals-utils-dataviewutils": "^0.3.0",
-    "powerbi-visuals-utils-formattingutils": "^0.2.2",
-    "powerbi-visuals-utils-interactivityutils": "^0.2.1",
-    "powerbi-visuals-utils-svgutils": "^0.2.1",
-    "powerbi-visuals-utils-tooltiputils": "^0.3.0",
-    "powerbi-visuals-utils-typeutils": "^0.2.0",
-    "powerbi-visuals-utils-chartutils": "^0.2.1",
-    "powerbi-visuals-utils-testutils": "^0.2.1",
-    "tslint": "4.0.2",
-    "typings": "2.0.0"
+    "karma-typescript-preprocessor": "0.3.1",
+    "powerbi-visuals-tools": "1.6.3",
+    "powerbi-visuals-utils-testutils": "1.0.0",
```

Comment:
No, this is `devDependencies`. Those libraries will not be included in result visual package. They use only in development environments such as testing or build process